src/main/java/com/microfocus/application/automation/tools/sv/runner/AbstractSvRunDescriptor.java

```diff
@@ -66,9 +68,14 @@ public SvServerSettingsModel[] getServers() {
         return SvServerSettingsGlobalConfiguration.getInstance().getServers();
     }
 
+    public boolean getHasConfigurePermission() {
+        return JenkinsUtils.hasItemConfigurePermission();
+    }
+
     @SuppressWarnings("unused")
     public ListBoxModel doFillServerNameItems() {
         ListBoxModel items = new ListBoxModel();
+            
```

Comment:
The previously incorrect permission checks distracted me from the actual problem: All this PR does is hide things from the UI. The security issue provides sample URLs you can access to see values returned while lacking all permissions except Overall/Read, and this PR currently does not address any of them.

 A permission check is needed here, and it'll be something like

```suggestion
            if (item == null || !item.hasPermission(Item.CONFIGURE)) {
                return items;
            }
```

Add `@AncestorInPath Item item` as a method parameter to make that work. Then apply the same to all other identified code locations.

---

I am not sure the rest of this PR even make sense for permission checks on the UI, as only users with the required permission can view the Configure forms. Could you explain?

Reply:
Hi @daniel-beck, 
The users with ExtendedRead permission can also view the Configure pages.
We decided to display a textbox instead of a dropdown list, if the user doesn't have Job/Configure permission, so that the java method _doFill..Items()_ will never be called.

**NOTE:** We'd like to use a global method accessible from config.jelly, from performance reasons because sometimes we have multiple dropdown controls and also javascript code that depends on this permission, like in https://github.com/jenkinsci/hpe-application-automation-tools-plugin/pull/658.

If you think is still necessary to change the _doFill..Items()_ methods, we can do that but would be redundant.
Please advise.

Reply:
>  The users with ExtendedRead permission can also view the Configure pages.

Good point, I thought the ExtendedRead view displayed only simplified read-only form fields and that they did not send requests.

> If you think is still necessary to change the doFill..Items() methods, we can do that but would be redundant.

It is necessary and not redundant, since they can be accessed directly via their URL, without going through the UI.

Reply:
Ok, thank you so much.
@alin-iordache-ot please update as suggested by Daniel.

Reply:
@dorin7bogdan Already did.